### PR TITLE
feat(core): form-alignment

### PR DESCRIPTION
**Pull request by fynnfeldpausch**

### CODE REVIEW

File: core/src/components/cat-input/cat-input.scss [hunk 1]
```diff
@@ -4,9 +4,8 @@
 @use 'utils/color';
 
 :host {
+  --input-height: 2.5rem;
```
Comment (fynnfeldpausch) [inline core/src/components/cat-input/cat-input.scss:7]: Why has this become a variable?

Comment (emmoldovan) [reply core/src/components/cat-input/cat-input.scss:7]: Because the min height of the label depends on the input height

Comment (fynnfeldpausch) [reply core/src/components/cat-input/cat-input.scss:7]: but the input height cannot be changed, right? If it is just about reusing the value, than we should use a private SCSS variable here. And also in the textarea. My question was, why is it a CSS custom property and not a SCSS variable.

Comment (emmoldovan) [reply core/src/components/cat-input/cat-input.scss:7]: Yes, you are right. It should have been a scss variable

Comment (fynnfeldpausch) [reply core/src/components/cat-input/cat-input.scss:7]: ok, please correct this (also in textarea, i guess) and then we can merge

**Pull request merged**